### ISSUE TITLE
Process Monitor Stop Commands Not Working

### ISSUE THREAD

**Issue by tjzhang1** [jtpereyda/boofuzz#445]
Hello,

I discovered this issue when the process_monitor_unix.py failed to restart a program that had just crashed. I got the following error:

 File "/home/debian/fuzzENV/lib/python2.7/site-packages/boofuzz/utils/process_monitor_pedrpc_server.py", line 210, in _stop_target
    os.system(command)
TypeError: system() argument 1 must be string, not list

The process_monitor_unix.py script failed to successfully stop my program because it didn't understand my stop command. When I wrote my fuzzing script, I set the start and stop commands with:
`
procmon.set_options(start_commands=["make start"], stop_commands=["TERMINATE_PID"])
`
According to function `_stop_target()` in [process_monitor_pedrpc_server.py](https://github.com/jtpereyda/boofuzz/blob/master/boofuzz/utils/process_monitor_pedrpc_server.py), the command "TERMINATE_PID" should tell the target process to stop automatically.

However, it seems that the script ignored the "TERMINATE_PID" and skipped straight to `os.system(command)`.

After some further digging, I found that `set_stop_commands()` uses `_split_command_if_str()` to save my stop commands as a list of strings, formatted in a way that is compatible with `subprocess.Popen()`. Which means that using `os.system()` to call my command is incorrect, because `os.system()` expects a string, not a list of strings, resulting in the error above.

I made the following changes to `_stop_target()` in my local directories.

1. Changed `if command == "TERMINATE_PID"` to `if command == ["TERMINATE_PID"]' because stop commands are stored as lists of strings
2. Changed `os.system(command)` to `subprocess.Popen(command)`
3. Added `import subprocess` to the top of process_monitor_pedrpc_server.py

This seemed to fix my problems, and the "TERMINATE_PID" command works as expected. I also noticed that the start commands are called via `subprocess.Popen()` too, which is why the process manager was able to start my program in the first place.

**TL;DR: The stop commands are formatted to be compatible with `subprocess.Popen()`, but the `_stop_target()` function incorrectly treats them as regular strings**

Thanks for reading, and keep up the good work!

**Comment by SR4ven** [comment]
Many thanks for providing such a detailed bug report and fix @tjzhang1!

I'll test the changes and open a PR.
